spec/infinispan_local_spec.js

```diff
@@ -5,370 +5,274 @@ var readFile = Promise.denodeify(require('fs').readFile);
 var f = require('../lib/functional');
 var t = require('./utils/testing'); // Testing dependency
 
-describe('Infinispan local client', function() {
-  var client = t.client(t.local);
+function describeInfinispanBehaviourInLocalModeWithDifferentHotrodProtocol(options) {
+  describe('with hotrod protocol: ' + options.version, function () {
+    var client = t.client(t.local, null, options.version);
 
-  beforeEach(function(done) { client
```

Comment:
@galderz jasmine-node has an argument - 
--config NAME VALUE- set a global variable in process.env

So  I can change the specs so that they are looking for process.env.VAR variable. If it exists then all specs will run with the same protocol version, otherwise 2.2 will be used as default. WDYT?